.gitignore

```diff
@@ -2,6 +2,7 @@ __pycache__/
 *.pyc
 *.json
 output/
+env-light/
 ctf_gym.egg-info/
```

Comment:
Good catch on adding `env-light` here. Could you add `env-full` as well?

Reply:
Will do! And I did not test this with the training code. I can do that later today and let you know if it works. I am assuming you're referencing the `train_against_easy.py` script?

Reply:
Yeah testing with that script would be great! I still haven't merged that PR. I'll mark you as a reviewer for it -- hope you don't mind.

Reply:
Just successfully tested the training script. Had to add a few changes to `base_policies.py` (no longer need to index the first element of `obs_batch`), and updated `.gitignore` to include `ray_test` output from trainings.